messaging/event.cc

```diff
@@ -6,17 +6,27 @@
 #include <exception>
 #include <filesystem>
 
-#include <sys/eventfd.h>
 #include <sys/mman.h>
 #include <sys/stat.h>
 #include <unistd.h>
 #include <poll.h>
 #include <signal.h>
 #include <fcntl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#ifdef __APPLE__
+#define ppoll(fds, nfds, timeout, sigmask) poll(fds, nfds, timeout != nullptr ? (timeout->tv_sec * 1000 + timeout->tv_nsec / 1000000) : -1)
```

Comment:
what's this for? wouldn't it hit an assert before this is even called?

Reply:
it would, but `wait`, `peek` and `wait_for_one` use that so i've added this stub for compilation to succeed

Reply:
otherwise would need couple of additional `#ifdef`s in the mentioned methods

Reply:
can you try to write this more like this one? https://github.com/commaai/cereal/blob/master/logger/logger.h

Reply:
Made it variadic macro to skip unused sigmask arg, not sure if that's what you meant

Reply:
I think i get it know, take a look at `PPOLL` now